src/app/vginventory/src/components/VideoGameEdit.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useUpdateVideoGame } from '../hooks/useUpdateVideoGame';
+
+export const VideoGameEdit = (props: any) => {
+    const { videogame, description, onDescriptionChange, onUpdate } = useUpdateVideoGame(props.location.state.game);
+    
+    return (
+        <>
+            <h1>{videogame.title}</h1>
+            <form>
+                <label>
+                    Description:
+                <textarea value={description} onChange={onDescriptionChange} />
+                </label>
+                <p><input type="submit" value="Update" onClick={onUpdate} /></p>
```

Comment:
When pressing Update, even though it does update the description, there is no indicator that something changed. After updating, can we have it go back to video game page so that you can see your updated description?